packages/web-react/src/types/textFieldBase.ts

```diff
@@ -13,7 +13,9 @@ export type SpiritTextFieldBaseProps = TextFieldBaseProps;
 
 export type TextFieldBaseInputProps = TextInputProps & TextFieldBaseMultiLineProps;
 
-export interface SpiritTextFieldBaseInputProps extends TextFieldBaseInputProps {}
+export interface SpiritTextFieldBaseInputProps extends TextFieldBaseInputProps {
+  id: string;
```

Comment:
Please, check the `input.ts` where you can find `InputBaseProps`. Maybe we can add the `id` there to be required by all `input` components. So we should keep the DRY principle.

Reply:
After mutual consultation, I modified the TextFieldElementBaseProps and TextAreaElementBaseProps types and also introduced a new RequiredId interface